### ISSUE TITLE
netest access to ergm$coef in statnet4

### ISSUE THREAD

**Issue by smjenness** [statnet/EpiModel#537]
warning, need to update code.
```r
Warning message:
Using x$coef to access the coefficient vector of an ergm is deprecated. Use coef(x) instead. 
```

```r
nw <- network_initialize(n = 500)
formation <- ~edges + concurrent + degrange(from = 4)
target.stats <- c(175, 110, 0)
coef.diss <- dissolution_coefs(dissolution = ~offset(edges), duration = 50)
est <- netest(nw, formation, target.stats, coef.diss)
```

**Comment by chad-klumb** [comment]
I believe this is resolved by 207ec6b0990878f9af3c0206e5bbf7c23fc94e22 but let me know if you have any more concerns about this issue.

**Comment by smjenness** [comment]
addressed above
